log/stdlib.go

```diff
@@ -49,6 +52,11 @@ func MessageKey(key string) StdlibAdapterOption {
 	return func(a *StdlibAdapter) { a.messageKey = key }
 }
 
+// Prefix sets value of the prefix for the actual log message. By default, there's no prefix.
```

Comment:
What does setting the prefix do?

Reply:
It allows client code to use the adapter for stdlib logger which is set by the prefix. There is an example in [the issue](https://github.com/go-kit/kit/issues/1035).

Reply:
Can you please add some more detail to the doc comment?

Reply:
sure. done